site/content/docs/06-server-side-rendering.md

```diff
@@ -2,19 +2,31 @@
 title: Server-side rendering
 ---
 
-Sapper, by default, renders server-side first (SSR), and then re-mounts any dynamic elements on the client. Svelte provides [excellent support for this](https://svelte.dev/docs#Server-side_component_API). This has benefits in performance and search engine indexing, among others, but comes with its own set of complexities.
+Sapper will render any component on the server side and send it to the client as HTML. It will then run the component again on the client side to allow it to update based on dynamic data. This is known as "hydration." Svelte has excellent support for this.
```

Comment:
"Svelte has excellent support for this." - Svelte supports it, but I'm not sure I'd say it's excellent. There's lots of issues around it - it's slow and doesn't work well with videos, iframes, etc.

Anyway, I think it's a bit late in the documentation to explain what hydration is because hydration is mentioned a few times before you get to this section

Reply:
I just kept "excellent support" from what was there before; I certainly don't mind deleting it. It doesn't help anyone.

I tried searching for hydration in the docs and I can't find any explanation of it. That's why I added it. Is it explain somewhere under a different name? The term is used twice before but without defining it.